### ISSUE TITLE
Custom services do not work if passed into the services property as an array

### ISSUE THREAD

**Issue by jdavis61** [webdriverio/webdriverio#3395]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.3.5
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 10.14.2
 - **NPM version:** 6.4.1
 - **Browser name and version:** Chrome 71
 - **Platform name and version:** Windows 10
 - **Additional wdio packages used (if applicable):** @wdio/cli: 5.3.5, @wdio/local-runner: 5.3.5, @wdio/mocha-framework: 5.3.2, @wdio/selenium-standalone-service: 5.2.2, @wdio/spec-reporter: 5.2.3, @wdio/sync: 5.3.2

**Config of WebdriverIO**
An example of how you configured WebdriverIO config
```
const MyCustomService = require('./build/index.js').default;
const FoodService = require('./build/index.js').foodService;

exports.config = {
...
services: [[FoodService, { food: 'Hamburger' }], 'selenium-standalone', [MyCustomService]],
...
};
```
Click [here](https://gist.github.com/jdavis61/885f827a27cea69b8923420ea68231e5#file-wdio-conf-js) to view the complete wdio.conf.js file.

**Describe the bug**
The @wdio/runner initialiseServices method does not pass the instantiated object to the initialisedServices return [array](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-runner/src/utils.js#L31). Instead the array from the services property is passed which causes the custom service not to be recognized by webdriverio.

For example passing the below CustomService to wdio using `services: [[MyCustomService]],` will generate the error `browser.testCommand() is not a command` when calling `browser.testCommand()`.

```
class MyCustomService {
    before() {
        browser.addCommand('testCommand', selector => $$(selector));
    };
}
export default new MyCustomService();
```

**To Reproduce**
This example adds two custom commands, `browser.elements` in [customService.js](https://gist.github.com/jdavis61/885f827a27cea69b8923420ea68231e5#file-customservice-js), another one named `getFavoriteFood` in [foodService.js](https://gist.github.com/jdavis61/885f827a27cea69b8923420ea68231e5#file-foodservice-js).
1. Download the reproducible example by running `git clone https://gist.github.com/885f827a27cea69b8923420ea68231e5.git test`.
2. cd test
3. npm run build.
4. npm run test

Executing the steps to reproduce will result in the errors: `browser.getFavoriteFood is not a function` and `browser.elements is not a function`.

**Expected behavior**
Expecting webdriverio to recognize the commands from my gist because the CustomService / FoodService is instantiated and follows the convention from the [documentation](https://webdriver.io/docs/customservices.html).

**Log**
Log information is found [here](https://gist.github.com/jdavis61/885f827a27cea69b8923420ea68231e5#file-log).

**Additional context**
I was able to fix the issue by changing lines [41 through 45 ](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-runner/src/utils.js#L41-L45) of the @wdio/runner utils.js file to the below code. If this is an appropriate solution please advise and I will open a PR.
```
     if (typeof serviceName === 'object') {
          log.debug(`initialise custom service "${serviceName}"`);
          if (Array.isArray(serviceName)) {
              if (serviceName.length === 2) {
                  var keys = Object.keys(serviceName[1]);
                  keys.forEach(key => {
                      serviceName[0][key] = serviceName[1][key];
                  });
              }
              initialisedServices.push(serviceName[0]);
          }
          initialisedServices.push(serviceName);
          continue;
      }
```

**Comment by abjerstedt** [comment]
@christian-bromann take a look at the suggest fix at the bottom (the additional context area) and determine if we can use this fix. This is also impacting my team.

**Comment by christian-bromann** [comment]
fixed in #3418
